api/routes.py

```diff
@@ -88,14 +88,15 @@ def getBikeCountData(id: int):
         else:
             return abort(404)
 
-
-    @api.route("/data/villo/<string:kind>")
-    @swag_from("swagger/get_villo_data.yml")
-    def getVilloData(kind):
+    # Historic data
+    @api.route("/api/data/historical/<string:kind>")
```

Comment:
1. The api endpoints don't start by `/api` anymore. |
2. No need for the /data in the URL (redundant)